src/Symfony/Component/Config/Resource/DirectoryResource.php

```diff
@@ -84,8 +84,15 @@ public function isFresh($timestamp)
                 continue;
             }
 
+            // for broken links
+            try {
+                $fileMTime = $file->getMTime();
```

Comment:
All PHP versions since `5.3`.

```php
symlink('/tmp/target_not_exists', '/tmp/link');
$file = new SplFileInfo('/tmp/link');
var_dump($file->getMTime());
```

https://3v4l.org/nLNYH